src/domain/sales-channels/tables/placeholder.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+
+import Button from "../../../components/fundamentals/button"
+import SidedMouthFaceIcon from "../../../components/fundamentals/icons/sided-mouth-face"
+
+function Placeholder({ showAddModal }) {
+  return (
+    <div className="h-full flex flex-col justify-center items-center">
+      <SidedMouthFaceIcon color="grey-50" width="48" height="48" />
```

Comment:
the color property here doesn't render the component correctly. An easy fix could be to remove the property and wrap in a span with the color property set 
```suggestion
      <span className="text-grey-50">
        <SidedMouthFaceIcon width="48" height="48" />
      </span>
```